### PR TITLE
Fix panic, in handle_block

### PR DESCRIPTION
**Pull request by vldm**
Closes https://github.com/exonum/exonum/issues/186

### CODE REVIEW

File: exonum/src/node/state.rs [hunk 1]
```diff
@@ -703,10 +703,16 @@ impl State {
     /// Transaction is ignored if the following criteria are fulfilled:
     /// - transactions pool size is exceeded
     /// - transaction isn't contained in unknown transaction list of any propose
-    pub fn add_transaction(&mut self, tx_hash: Hash, msg: Box<Transaction>) -> Vec<(Hash, Round)> {
+    /// - transaction isn't a part of block
+    pub fn add_transaction(
+        &mut self,
+        tx_hash: Hash,
+        msg: Box<Transaction>,
+        // if tx is in some of propose or in a block,
```
Comment (DarkEld3r) [inline exonum/src/node/state.rs]: Maybe move that comment to the function description?

Comment (vldm) [inline exonum/src/node/state.rs]: For me  it's more clear, with rustfmt `block` mode, when the comment is as close as possible with the place that it comments.

File: exonum/src/blockchain/mod.rs [hunk 2]
```diff
@@ -221,7 +221,9 @@ impl Blockchain {
             let last_hash = self.last_hash();
             // Save & execute transactions
             for (index, hash) in tx_hashes.iter().enumerate() {
-                let tx = &pool[hash];
+                let tx = pool.get(hash).expect(
+                    "BUG: Cannot found transaction in pool.",
```
Comment (gisochre) [inline exonum/src/blockchain/mod.rs]: find

File: exonum/src/node/state.rs [hunk 3]
```diff
@@ -703,10 +703,16 @@ impl State {
...
+        // we should add it, or we can stuck on some state
```
Comment (gisochre) [inline exonum/src/node/state.rs]: could become

Comment (gisochre) [inline exonum/src/node/state.rs]: in

**Pull request merged**